src/main/java/com/microsoft/graph/exceptions/Error.java

```diff
@@ -0,0 +1,81 @@
+package com.microsoft.graph.exceptions;
+
+public class Error {
```

Comment:
I was under the understanding that this could later be used to handle multiple types of errors aside from just Authentication errors. If were using it for OData response bodies would it still make sense to rename it to AuthenticationError?

Reply:
Got it. If it is intended to be used for OData responses as well, then leave as is.